packages/editor-ui/src/components/AssignmentCollection/Assignment.vue

```diff
@@ -56,6 +56,21 @@ const assignmentTypeToNodeProperty = (
 	}
 };
 
+function getExpressionResult(value: string): Result<unknown, unknown> {
```

Comment:
Does it really need to be this return type? (`Result<unknown, unknown>`)
I know a lot of underlying code has quite bad typing already but with this we're just introducing more bad typing

I saw later you cast an error to `Error` type (`result.error as Error`) and `resolveExpression` could return almost anything (and that should be also fixed but I don't want you to go down that rabbit hole) but at least the above could be `Result<unknown, Error>`

Reply:
You're right about `Error`, but I think `unknown` for the first type argument is the best we can do since the result of an expression can be any value

Reply:
Yes, I have seen there are a couple of type checks in `stringifyExpressionResult`